tests/provider/dwd/radar/test_api_historic.py

```diff
@@ -247,6 +247,7 @@ def test_radar_request_composite_historic_radolan_rw_timerange():
     # TODO: Verify data.
 
 
+@pytest.mark.xfail
```

Comment:
Again? I thought I had fixed all relevant places. If it will be out of service, let's add `reason="Out of service"` here. If it will always fail, we should also add `strict=True`.

If it is something more of a soft failure, we should `raise pytest.skip(reason="Foobar")` from within the function, right? This also applies to the other occurences.